sdk/search/search-documents/test/utils/recordedClient.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+import * as path from "path";
+
+import { env, RecorderEnvironmentSetup } from "@azure/test-utils-recorder";
+import { isNode } from "@azure/core-http";
+
+import { AzureKeyCredential, SearchIndexClient, SearchServiceClient } from "../../src/index";
+
+if (isNode) {
+  dotenv.config({ path: path.join(__dirname, "..", ".env") });
+}
+
+export interface Clients<IndexModel> {
+  indexClient: SearchIndexClient<IndexModel>;
+  serviceClient: SearchServiceClient;
+}
+
+const replaceableVariables: { [k: string]: string } = {
+  SEARCH_API_ADMIN_KEY: "admin_key",
+  SEARCH_API_ADMIN_KEY_ALT: "admin_key_alt",
+  ENDPOINT: "https://endpoint/"
```

Comment:
```suggestion
  ENDPOINT: "https://endpoint"
```

Maybe this would fix the issue.

Reply:
This worked!!! ❤️